storages/response_handler.go

```diff
@@ -122,19 +116,23 @@ func (rm *responseMerger) createResponse(successes []transport.ResErrTuple) (res
 	}
 	oContainer := objectsContainer{
 		list: make([]ObjectInfo, 0),
-		set:  make(map[string]struct{}, 0),
+		set:  make(map[string]struct{}),
```

Comment:
Please avoid using plain strings as a key. What are they?. Use type aliasing to give them proper domain name.

https://testing.googleblog.com/2017/11/obsessed-with-primitives.html
http://wiki.c2.com/?PrimitiveObsession
http://wiki.c2.com/?StringlyTyped